app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str):
+    command_list = command.split(" ")
+    path_out_list = command_list[2].split("/")
+    path = ""
+    if len(path_out_list) > 1:
+        for i in range(len(path_out_list) - 1):
+            path = os.path.join(path_out_list[i])
```

Comment:
```suggestion
            path = os.path.join(path, path_out_list[i])
```
You should add next item to the part that is already stored in `path`

Reply:
1. Запушил ошибочно последний коммит  _Solution3 не могу удалить.
2. Перепроверил и обнаружил удивительную вещь os.path.join некорректно работает у меня в винде (в этом проекте), при попытке создать новое дерево в корне диска D. Создает путь/дерево в текущем каталоге.  D: (с двоеточием вообще игнорирует). Корректно работает только код первого коммита.